packages/arb-token-bridge-ui/src/token-bridge-sdk/EthDepositStarter.ts

```diff
@@ -68,12 +73,18 @@ export class EthDepositStarter extends BridgeTransferStarter {
 
   public async approveNativeCurrency({
     signer,
-    amount
+    amount,
+    options
   }: ApproveNativeCurrencyProps) {
     const ethBridger = await this.getBridger()
+
+    const amountToApprove = amount.add(
+      options?.approvalAmountIncrease || BigNumber.from(0)
```

Comment:
```suggestion
      options?.approvalAmountIncrease ?? constants.Zero
```

if it's `undefined`, it will go to the ?? case, so we can just use that instead of ||?

Reply:
updated